packages/core/test/lastfm.test.ts

```diff
@@ -73,3 +73,23 @@ test('search tracks', async t => {
 //   t.true(data.results.trackmatches.track instanceof Array);
 //   t.true(data.results.trackmatches.track.length > 0);
 // });
+
```

Comment:
Can you also add a test for fetching more than 1000 favorites? And mock the api access?